src/verifier.ts

```diff
@@ -216,6 +217,6 @@ export interface VerifierOptions {
   timeout?: number;
   tags?: string[];
   monkeypatch?: string;
-  format?: 'json' | 'RspecJunitFormatter';
```

Comment:
@TimothyJones may bad. I thought this was breaking as it was replaced by `xml`. It looks OK now, so maybe I'm going crazy.